### DISCUSSION TITLE
你好，我想问问这两人的命运如何？

### DISCUSSION THREAD

**Discussion by TaoTie2049** [SylarLong/iztro#153] [Q&A]
![4c620abbe9c7dd67bb2e5b9dd1a7c78d](https://github.com/SylarLong/iztro/assets/154208202/ac35fdbd-484c-4dac-bfcc-309119fe7af6)
![6eba70aba22f32cfb00e4249cdc15c88](https://github.com/SylarLong/iztro/assets/154208202/01cc1cd3-4d73-440f-8af5-5dbfd9dfd75f)

**Comment by SylarLong** [comment]
抱歉，现在已经没有在线解盘的活动了 #151
